cmd/svcat/output/class.go

```diff
@@ -77,15 +76,21 @@ func WriteClass(w io.Writer, outputFormat string, class servicecatalog.Class) {
 }
 
 // WriteClassDetails prints details for a single class.
-func WriteClassDetails(w io.Writer, class *v1beta1.ClusterServiceClass) {
+func WriteClassDetails(w io.Writer, class servicecatalog.Class) {
+	scope := getScope(class)
+	spec := class.GetSpec()
 	t := NewDetailsTable(w)
 	t.AppendBulk([][]string{
-		{"Name:", class.Spec.ExternalName},
-		{"Description:", class.Spec.Description},
-		{"UUID:", string(class.Name)},
-		{"Status:", getClassStatusText(class.Status)},
-		{"Tags:", strings.Join(class.Spec.Tags, ", ")},
-		{"Broker:", class.Spec.ClusterServiceBrokerName},
+		{"Name:", spec.ExternalName},
+		{"Description:", spec.Description},
+		{"UUID:", string(class.GetName())},
+		{"Status:", class.GetStatusText()},
+		{"Tags:", strings.Join(spec.Tags, ", ")},
+		{"Broker:", class.GetServiceBroker()},
+		{"Scope:", scope},
 	})
+	if class.GetNamespace() != "" {
```

Comment:
I realize this seems like a total nit but It would be preferable for namespace to be listed immediately after name. That way related data is near each other in the output and this matches what people expected based on experience with kubectl.

Same for scope, it's more important than the status or the tags, etc.

See https://github.com/kubernetes-incubator/service-catalog/issues/2215 for the desired order of the fields, but the summary is: Name, [Namespace], Scope.